cmd/up.go

```diff
@@ -70,20 +69,34 @@ func executeUp(devPath string) error {
 		return err
 	}
 
-	err = storage.Insert(namespace, d.Name, dev.Swap.Deployment.Container, sy.LocalPath, sy.GUIAddress)
+	err = storage.Insert(namespace, name, dev.Swap.Deployment.Container, sy.LocalPath, sy.GUIAddress)
 	if err != nil {
+		if err == storage.ErrAlreadyRunning {
+			return fmt.Errorf("there is already an entry for %s. Are you running 'cnd up' somewhere else?", fullname)
+		}
+
 		return err
 	}
 
-	defer stop(sy, pf)
-	err = pf.Start(client, restConfig, pod)
-	return err
+	channel := make(chan os.Signal, 1)
+	signal.Notify(channel, os.Interrupt)
+	go func() {
+		<-channel
+		stop(sy, pf)
+		return
+	}()
+
+	return pf.Start(client, restConfig, pod)
 }
 
 func stop(sy *syncthing.Syncthing, pf *forward.CNDPortForward) {
+	fmt.Println()
```

Comment:
this code only executes when you do ctrl+c, is just so the logs start on a new line
![image](https://user-images.githubusercontent.com/475313/49533482-efa07d00-f873-11e8-9a6b-14552c26503f.png)

Instead of in a weird half line
![image](https://user-images.githubusercontent.com/475313/49533492-ff1fc600-f873-11e8-8ade-b99ea9aa3488.png)